src/renderers/dom/client/utils/__tests__/setInnerHTML.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+var setInnerHTML = require('setInnerHTML');
+
+describe('setInnerHTML', function() {
+  describe('when the node has innerHTML property', () => {
+    it('sets innerHTML on it', function() {
+      var node = document.createElement('div');
+      var html = '<h1>hello</h1>';
+      setInnerHTML(node, html);
+      expect(node.innerHTML).toBe(html);
+    });
+  });
+
+  // SVGElements on IE don't have innerHTML
+  describe('when the node does not innerHTML property', () => {
+    it('sets innerHTML on it', function() {
+      var node = document.createElement('svg');
+      delete node.innerHTML;
```

Comment:
You're right - I knew it wouldn't work in the browser but was hoping that it would work under the fake JSDOM environment and I meant to confirm, but missed out.

I'm kinda stuck for how to accurately test this then, apart from creating a complete mock node like

```
var node = { appendChild: spy() };
expect(node.appendChild).toHaveBeenCalled();
```

Seems like a bit of a pointless test though if I'm not even passing in a real node.

JSDOM also seems to lack `SVGElement` which makes it slightly harder as well